docs/test-cases.md

```diff
@@ -57,3 +57,13 @@
    - Click on the canvas closes breakpoints selector
    - Styles added on each breakpoint are only applied to each breakpoint, verify that by resizing the canvas in preview mode
    - Resizing canvas width in preview mode doesn't limit resize slider to each breakpoint
+
+1. Auth
```

Comment:
the format of those should be step by step: click this click that, you know, like you are explaining to a person who knows nothing about development, because this might be actually the case once we ask someone to help test it

Reply:
But then this is not a testing case because a testing case is for a QA and what needs to be tested

Reply:
you mean we need to rename testing-cases into something else? Sure lets do it. I just think we should be clear about how things are reproduced because they may be not trivial, especially when you have less context than you do right now

Reply:
oh, I think I missunderstood what you said above. Made things more clear